web/packages/build/eslint.config.mjs

```diff
@@ -242,5 +242,36 @@ export default tseslint.config(
         },
       ],
     },
+  },
+
+  /*
+   * Restricted hook imports
+   *
+   * We already have our own, Electron and browser safe hooks, so we don't want any usage of the equivalent
+   * hooks from `usehooks-ts`
+   */
+  {
+    files: ['e/web/**/*.{ts,tsx,js,jsx}', 'web/packages/**/*.{ts,tsx,js,jsx}'],
+    rules: {
+      'no-restricted-imports': [
```

Comment:
This seems to have overwritten `no-restricted-imports` set by earlier rules making them ineffective. For example, this line returns a warning about an unused directive:

https://github.com/gravitational/teleport/blob/5743c4bd83fad11aab0d97a5d222ec12b8e94a89/web/packages/shared/components/UnifiedResources/UnifiedResources.tsx#L59-L60

I know that ESLint has some complex rules regarding how options are merged, but off the top of my head idk how to fix this.

Reply:
> When more than one configuration object specifies the same rule, the rule configuration is merged with the later object taking precedence over any previous objects.

https://eslint.org/docs/latest/use/configure/rules#using-configuration-files

Reply:
oh.. oops, how annoying

Reply:
I'll figure it out and enable `reportUnusedDisableDirectives` so that we don't miss this in the future.